### ISSUE TITLE
Armenian & Georgian

### ISSUE THREAD

**Issue by thisismattmiller** [lcnetdev/scriptshifter#6]
Two characters that didn't transliterate (The punctuation mark ՝ ("Armenian comma") did not transliterate)

**Comment by scossu** [comment]
U+0571 (ձ) and U+055D ( ՝) are missing from table. 

U+0575 (յ): this will require a new table, e.g. `armenian_modern.yml` inheriting from `armenian.yml` and containing the override. 

U+10E4 (ფ): the problem is at https://github.com/lcnetdev/scriptshifter/blob/main/scriptshifter/tables/data/georgian.yml#L192

U+10E5 (ქ): the problem is at https://github.com/lcnetdev/scriptshifter/blob/main/scriptshifter/tables/data/georgian.yml#L193

**Comment by scossu** [comment]
U+0571 and U+055D have been resolved. 

@rrroche do you want to take a crack at the `armenian_modern` table? I can walk you through the process if needed.

**Comment by rrroche** [comment]
Yes, I can try it tomorrow and let you know if and where I get stuck. Thanks for the idea!

**Comment by rrroche** [comment]
Is there a way to add a conditional for whether something is the first letter? As in, some sort of check for if a proceeding character is absent, a space, or a punctuation mark?

**Comment by scossu** [comment]
That would require writing a hook. I'll take care of that if you can specify the behavior you need.

**Comment by rrroche** [comment]
Յ and յ only become Ḥ and ḥ when they are the start of a word or stem in a compound.

**Comment by scossu** [comment]
And what do they become when they are not at the beginning?

**Comment by scossu** [comment]
I deleted my earlier comment because I figured out the answer. If you have started some work on modern Armenian, can you open a pull request so I can add the hook part to it?

**Comment by rrroche** [comment]
I talked to Brigita, the Armenian cataloger, about this, and she wanted to talk to other Armenian language catalogers about whether they even want the option to add the variations in an automated way. What you already have covers the most commonly used modern Armenian, and the transliteration variations may only be used in pretty limited circumstances. Their next meeting would be in January. Is that too long to wait?

**Comment by scossu** [comment]
It's not too long, but implementing the variants in any case shouldn't be too bad. Unless there is a chance that the addition would be more of an issue than a benefit, in which case I can hold off.

**Comment by scossu** [comment]
@rrroche just reviewing this ticket, sorry for the delay—Did the January meeting with the Armenian catalogers yield any interesting results? I can either put this issue on ice or work on it for Phase 2.

**Comment by rrroche** [comment]
I think we can put this issue on ice for now and I will put in a new ticket if we decide to go forward with it. Brigita just let me know she is on a detail until June, and so will not be able to bring it up at the International Association of Armenian Librarians and Archivists cataloging committee meeting until then.
